Source/PlasticSourceControl/Private/PlasticSourceControlUtils.cpp

```diff
@@ -32,6 +32,11 @@
 namespace PlasticSourceControlUtils
 {
 
+// 11.0.16.7608 add support for history --limit. It displays the N last revisions of the specified items.
+// https://www.plasticscm.com/download/releasenotes/11.0.16.7608
+static const FSoftwareVersion s_NewHistoryLimitPlasticScmVersion(TEXT("11.0.16.7608"));
```

Comment:
We now have:

PlasticSourceControlUtils
- s_NewHistoryLimitPlasticScmVersion(TEXT("11.0.16.7608"))

PlasticSourceControlOperations
- s_NewChangelistFileArgsPlasticScmVersion(TEXT("11.0.16.7248"))

SoftwareVersion
- s_OldestSupportedPlasticScmVersion(TEXT("9.0.16.4839"))

It'd be great to have those 3 constants extracted to the same utility class. Ideally, a specific class to cover just features by version.

Reply:
Thanks, good catch, I refactored it yesterday